pages/grants.js

```diff
@@ -129,158 +129,140 @@ export default function Grants({ posts, categories, types, search }) {
-                  <p>
-                    <Link href="/grants/apprenticeships">Apprenticeships</Link>{" "}
-                    are largely about building a relationship between apprentice
-                    and mentor
-                  </p>
-                </li>
-              </ul>
 
-              <p className="mb-8">
-                Whether you're gearing up to submit your proposal or just
-                thinking about it, the best way to get started is to join the
-                Foundation group on Urbit:
-              </p>
+            <h2 id="join-community" className="pb-12">
+              Community
+            </h2>
 
-              <JoinGroup
-                emphasize
-                className="mb-8"
-                groupName="~wolref-podlex/foundation"
-              />
+              <div className="flex flex-column flex-wrap md:flex-nowrap grid-cols-2 gap-x-12  pb-12">
 
-              <p>
-                If you're not on the network, reach out to us at{" "}
-                <a href="mailto:grants@urbit.org">grants@urbit.org</a> and we'll
-                get back to you within a couple of days.
-              </p>
-            </div>
+              <div className="md:basis-1/2">
+
+                <p className="mb-4">
+                  We believe that grants are more successful when tackled with the
+                  support of a community, and community starts with individual
+                  relationships.
+                </p>
+                <ul className="mb-4">
+                  <li className="mb-2">
+                    <p>
+                      <Link href="/grants/proposals">Proposals</Link> are approved
+                      and stewarded by past grant workers
+                    </p>
+                  </li>
+                  <li className="mb-2">
+                    <p>
+                      <Link href="/grants/bounties">Bounties</Link> are overseen
+                      by those that post them
+                    </p>
+                  </li>
+                  <li className="mb-2">
+                    <p>
+                      <Link href="/grants/apprenticeships">Apprenticeships</Link>{" "}
+                      are largely about building a relationship between apprentice
+                      and mentor
+                    </p>
+                  </li>
+                </ul>
+              </div>
+
+
+              <div className="md:basis-1/2">
+                <p className="mb-8">
+                  Whether you're gearing up to submit your proposal or just
+                  thinking about it, the best way to get started is to join the
+                  Foundation group on Urbit:
+                </p>
+
+                <JoinGroup
+                  emphasize
+                  className="mb-8"
+                  groupName="~wolref-podlex/foundation"
+                />
+
+                <p>
+                  If you're not on the network, reach out to us at{" "}
```

Comment:
Shouldn't we point them to the Getting Started page instead?

Reply:
Yeah, if you're getting grant money you should probably be on the network.  Good point.